riotfile.py

```diff
@@ -360,7 +360,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                         "celery": [
                             "~=5.0.5",
                             "~=5.0",  # most recent 5.x
```

Comment:
won't this still install it? do we need to remove this one too?

Reply:
good catch! looks like it would also install it